restructured-document/product-use.tex

```diff
@@ -0,0 +1,12 @@
+\section{Product use}
+
+\subsection{Target group}
+%seems incomplete...
+Groups of users with access to remote computers who need to speed up the processing time of their tasks.
+
+\subsection{Working conditions}
+\begin{itemize}
+\item A perpetual connection to a LAN.
+\item Hard disk space.
```

Comment:
What are you trying to say here?
Virtually every computer on the planet has hard disk space.